### ISSUE TITLE
[Feature Request] expose API for modifying configurations in standalone mode

### ISSUE THREAD

**Issue by frankli0324** [microsoft/monaco-editor#4095]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

here's the background of this request: https://stackoverflow.com/questions/76738291/update-monaco-editor-configurations-in-standalone-browser-editor

in short, I'm finding a way to do this properly:

```ts
import { IConfigurationService } from "monaco-editor/esm/vs/platform/configuration/common/configuration";
import { StandaloneServices } from "monaco-editor/esm/vs/editor/standalone/browser/standaloneServices";

(()=>{
  StandaloneServices.get(IConfigurationService).updateValue(
    "editor.maxTokenizationLineLength", 40000
  );
})();
```

**Comment by frankli0324** [comment]
should I stay with this non-public api?
